src/platform.c

```diff
@@ -566,6 +582,21 @@ static const char* kSokolFragSource =
 "  float4 col = val & mask ? float4(0.694, 0.686, 0.659, 1.0) : float4(0.192, 0.184, 0.157, 1.0);\n"
 "  return col;\n"
 "}\n";
+#else
+// GLSL
+"#version 410\n"
+"uniform sampler2D tex;\n"
+"in vec2 uv;\n"
+"out vec4 frag_color;\n"
+"void main() {\n"
+"  int x = int(uv.x * 400);\n"
+"  int y = int(uv.y * 200);\n"
```

Comment:
Maybe this should be `* 240`, not `* 200` :)

Reply:
Damn! I need new glasses lol. Looks good now.